test/specs/apexReplayDebugger.e2e.ts

```diff
@@ -59,6 +59,7 @@ describe('Apex Replay Debugger', async () => {
     // Reloading the window forces the extensions to be reloaded and this seems to fix
     // the issue.
     await utilities.runCommandFromCommandPrompt(workbench, 'Developer: Reload Window', 30);
+    await testSetup.verifyAllExtensionsAreRunning();
```

Comment:
After reloading the VSCode window, the Salesforce extensions need time to activate before they are used.  Adding this line verifies that all extensions are activated after the reload.